public/app/features/logs/logsModel.ts

```diff
@@ -789,3 +790,24 @@ function getIntervalInfo(scopedVars: ScopedVars, timespanMs: number): { interval
     return { interval: '$__interval' };
   }
 }
+
+/**
+ * Converts log row into data frame with a single row.
+ *
+ * @export
+ * @param {LogRowModel} logRow
+ * @return {DataFrame}
```

Comment:
I don't think this is needed, as it just repeats what is in function name + types.

Reply:
Changed in c880bb3b818bbedfba832fc0a3594503fb62ec3c